README.md

```diff
@@ -1,87 +1,142 @@
 # GitHub Actions for deploying to Azure
 
-## Automate your GitHub workflows using Azure Actions
-
 [GitHub Actions](https://help.github.com/articles/about-github-actions) gives you the flexibility to build an automated software development lifecycle workflow.
```

Comment:
This doc should focus on the login action. Background information and other actions are less important.

Reply:
Updated. Update the title and move GitHub Action and other actions' description to the reference section.